sysreqs/pkgconfig.json

```diff
@@ -3,7 +3,9 @@
     "sysreqs": "/pkg(:-|)config/",
     "platforms": {
       "DEB": "pkg-config",
+      "PKGBUILD": "",
```

Comment:
This seems to be a typo here.

Reply:
Thanks for catching those! Should be fixed now..